core/base/send-webhooks-sa.yaml

```diff
@@ -27,3 +27,15 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: send-webhooks
```

Comment:
can we include this role here: 
https://github.com/thoth-station/thoth-application/blob/357ec964b805cf15705fff91e6e4b0b402b65d81/core/overlays/test/argo-management.yaml#L16

and  remove it individual occurrence from overlays:
- https://github.com/thoth-station/thoth-application/blob/357ec964b805cf15705fff91e6e4b0b402b65d81/core/overlays/test/argo-management.yaml#L16
- https://github.com/thoth-station/thoth-application/blob/357ec964b805cf15705fff91e6e4b0b402b65d81/core/overlays/ocp4-stage/backend-stage/argo-management.yaml#L11